app/screens/wallet/TwelveWordsBackup.js

```diff
@@ -0,0 +1,180 @@
+// @flow
+import { clipboard } from 'electron';
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import type { RouterHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+// import { backupWallet } from '/redux/wallet/actions';
+import get from 'lodash.get';
+import { SmButton } from '/basicComponents';
+import { smColors } from '/vars';
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  flex-direction: flex-start;
+  padding: 50px;
+`;
+
+const Header = styled.span`
+  font-size: 31px;
+  font-weight: bold;
+  line-height: 42px;
+  color: ${smColors.lighterBlack};
+  margin-bottom: 20px;
+`;
+
+const HeaderExplanation = styled.div`
+  font-size: 16px;
+  color: ${smColors.lighterBlack};
+  line-height: 30px;
+  margin-bottom: 22px;
+`;
+
+const BaseText = styled.span`
+  font-size: 16px;
+  font-weight: normal;
+  line-height: 22px;
+  color: ${smColors.lighterBlack};
+`;
+
+const ActionLink = styled(BaseText)`
+  user-select: none;
+  color: ${smColors.darkGreen};
+  cursor: pointer;
+  &:hover {
+    opacity: 0.8;
+  }
+  &:active {
+    opacity: 0.6;
+  }
+`;
+
+const Text = styled.span`
+  font-size: 18px;
+  color: ${smColors.darkGray};
+  line-height: 24px;
+`;
+
+const Indices = styled(Text)`
+  color: ${smColors.darkGray50Alpha};
+  margin-right: 50px;
+`;
+
+const TwelveWordsContainer = styled.div`
+  border: 1px solid ${smColors.darkGreen};
+  padding: 28px;
+  margin-bottom: 22px;
+  column-count: 3;
+`;
+
+const WordWrapper = styled.div`
+  height: 50px;
+  line-height: 50px;
+`;
+
+const NotificationSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 48px;
```

Comment:
no need to set height here

Reply:
If height is not set, the buttons are pushed downwards when the mnemonic is copied. We talked about keeping the buttons fixed. Has this changed?

Reply:
ok then round it to 50px

Reply:
ok